taskfiles/Install.yml

```diff
@@ -0,0 +1,70 @@
+version: '3'
+
+tasks:
+  brew-install-jq:
+    internal: true
+    platforms: [darwin]
+    cmds: ["which jq > /dev/null || brew install jq"]
+    status:
+      - test -n "command -v jq"
+    preconditions:
+      - sh: 'which brew'
+        msg: '"brew" needed to install "jq"- see https://brew.sh'
+
+  brew-install-terraform:
+    internal: true
+    platforms: [darwin]
+    status:
+      - test -n "command -v terraform"
+    cmds:
+      - echo "Installing terraform..."
+      - brew tap hashicorp/tap && brew install hashicorp/tap/terraform
+      - echo -e "\nSuccess! 'terraform' installed."
+    preconditions:
+      - sh: 'which brew'
+        msg: '"brew" needed to install "Terraform" - see https://brew.sh'
+
+  go-install-changie:
+    desc: go install "changie"
+    internal: true
+    status:
+      - test -n "command -v changie"
+    cmds:
+      - task: go-install-tool
+        vars: { GO_TOOL: "changie", GO_TOOL_PATH: "github.com/miniscruff/changie@latest" }
+
+  go-install-gofumpt:
+    desc: go install "gofumpt"
+    internal: true
+    status:
+      - test -n "command -v gofumpt"
+    cmds:
+      - task: go-install-tool
+        vars: { GO_TOOL: "gofumpt", GO_TOOL_PATH: "mvdan.cc/gofumpt@latest" }
+
+  go-install-golangci-lint:
+    desc: go install "golangci-lint"
+    internal: true
+    status:
+      - test -n "command -v golangci-lint"
+    cmds:
+      - task: go-install-tool
+        vars: { GO_TOOL: "golangci-lint", GO_TOOL_PATH: "github.com/golangci/golangci-lint/cmd/golangci-lint@latest" }
+
+  go-install-tool:
+    desc: go install '{{.GO_TOOL}}' and set GOBIN if not set
+    internal: true
+    silent: true
+    vars:
+      IS_TOOL_INSTALLED:
+        sh: which {{.GO_TOOL}} > /dev/null || echo "1"
```

Comment:
This is what `status` is for - it would also greatly simplify the `cmds` section

Reply:
💯 - will update